### PR TITLE
{lang}[foss/2022a] R v4.2.1

### PR DESCRIPTION
**Pull request by branfosj**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by branfosj** [comment]
@boegelbot please test @ jsc-zen2

**Comment by branfosj** [comment]
Test report by @branfosj
**FAILED**
Build succeeded for 18 out of 21 (1 easyconfigs in total)
bear-pg0105u36b.bear.cluster - Linux RHEL 8.6, x86_64, Intel(R) Xeon(R) Platinum 8360Y CPU @ 2.40GHz (icelake), Python 3.6.8
See https://gist.github.com/c23177e43e931bad8f412dd413404e88 for a full test report.

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 2 out of 2 (1 easyconfigs in total)
bear-pg0105u36b.bear.cluster - Linux RHEL 8.6, x86_64, Intel(R) Xeon(R) Platinum 8360Y CPU @ 2.40GHz (icelake), Python 3.6.8
See https://gist.github.com/5f7455d161aecfa68584aab3db6d5aad for a full test report.

**Comment by SebastianAchilles** [comment]
Test report by @sebastianachilles
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
bdw-opensuse-154 - Linux openSUSE Leap 15.4 Alpha, x86_64, Intel(R) Core(TM) i7-6900K CPU @ 3.20GHz (broadwell), Python 3.6.15
See https://gist.github.com/96e5c52a8b476a2d1c0b205044366dc2 for a full test report.

**Comment by SebastianAchilles** [comment]
@boegelbot please test @ generoso

**Review by boegel — DISMISSED** [review]
lgtm (test report coming up)

**Comment by verdurin** [comment]
It's maybe worth mentioning here that it appears there's a problem with `Seurat` in that it and/or some of the projects that build on it need `HDF5` `<=1.8.x`. I know it's in `Bioconductor`, but obviously that builds on what's here.

This is the upstream bug report that was mentioned to us:

https://github.com/mojaveazure/seurat-disk/issues/30

Happy to open it as an issue - just wanted to mention here too.

**Comment by jfgrimm** [comment]
I'm seeing:
```
4366941 automake found. Running autogen.sh.                                             
4366942 + libtoolize --copy                                                             
4366943 autogen.sh: line 43: libtoolize: command not found                              
4366944 make --directory=libuv \                                                        
4366945         HAVE_DTRACE=0                    
4366946 make[1]: Entering directory `/dev/shm/jfg508/R/4.2.1/foss-2022a/httpuv/httpuv/src/libuv'
4366947 CDPATH="${ZSH_VERSION+.}:" && cd . && /bin/sh /dev/shm/jfg508/R/4.2.1/foss-2022a/httpuv/httpuv/src/libuv/missing aclocal-1.16 -I m4
4366948 /dev/shm/jfg508/R/4.2.1/foss-2022a/httpuv/httpuv/src/libuv/missing: line 81: aclocal-1.16: command not found            
4366949 WARNING: 'aclocal-1.16' is missing on your system.                              
4366950          You should only need it if you modified 'acinclude.m4' or              
4366951          'configure.ac' or m4 files included by 'configure.ac'.                 
4366952          The 'aclocal' program is part of the GNU Automake package:             
4366953          <https://www.gnu.org/software/automake>                                
4366954          It also requires GNU Autoconf, GNU m4 and Perl in order to run:        
4366955          <https://www.gnu.org/software/autoconf>                                
4366956          <https://www.gnu.org/software/m4/>                                     
4366957          <https://www.perl.org/>                                                
4366958 make[1]: *** [aclocal.m4] Error 127                                             
4366959 make[1]: Leaving directory `/dev/shm/jfg508/R/4.2.1/foss-2022a/httpuv/httpuv/src/libuv'
4366960 make: *** [libuv/.libs/libuv.a] Error 2                                         
4366961 ERROR: compilation failed for package httpuv
```
I guess we should add `('Autotools', '20220317')` as a `builddependency`

File: easybuild/easyconfigs/r/R/R-4.2.1-foss-2022a.eb [hunk 1]
```diff
@@ -0,0 +1,3359 @@
...
+toolchain = {'name': 'foss', 'version': '2022a'}
+
+source_urls = ['https://cloud.r-project.org/src/base/R-%(version_major)s']
+sources = [SOURCE_TAR_GZ]
+checksums = ['4d52db486d27848e54613d4ee977ad952ec08ce17807e1b525b10cd4436c643f']
+
+builddependencies = [
+    ('pkgconf', '1.8.0'),
+    ('Xvfb', '21.1.3'),
+]
```
Comment (jfgrimm) [inline easybuild/easyconfigs/r/R/R-4.2.1-foss-2022a.eb:17]: required for (at least) `httpuv`
```suggestion
    ('Autotools', '20220317'),
]
```

**Review by jfgrimm — CHANGES_REQUESTED** [review]

**Comment by boegel** [comment]
> 

@verdurin Please open a separate issue for this, we've been using HDF5 1.10.x for quite some time now, so this is definitely not a reason to go back to using HDF5 1.8.x (especially since the problem is specific to Seurat).

**Comment by boegel** [comment]
Going in, thanks @branfosj!

**Pull request merged**